README.md

```diff
@@ -1,4 +1,4 @@
-
+![Serial Keel Build](https://github.com/torsteingrindvik/serial-keel/.github/workflows/basics.yml/badge.svg)
```

Comment:
We'll know for sure whether this works only after merge :(

Reply:
That's ok. I like the idea though :slightly_smiling_face: 
If it fails for some reason after merge we can just force push that commit in `main` until it works.